wsgi/odyssy/odyssy/urls.py

```diff
@@ -18,5 +18,6 @@
 
 urlpatterns = [
     url(r'^admin/?', include(admin.site.urls)),
-    url(r'^', include('basic.urls'))
+    url(r'^', include('basic.urls')),
+    url(r'announcement/', include('announcement.urls'))
```

Comment:
I think there is problem in regex. It should be `^announcement/`. The regex mentioned in code matched with `<some gibberish>announcement/`. For example, it matches with `abcbannouncemnt/`